packages/core/src/fhir-deduplication/__tests__/group-same-encounters.test.ts

```diff
@@ -14,20 +14,26 @@ import {
 import { groupSameEncounters } from "../resources/encounter";
 import { dateTime, dateTime2 } from "./examples/condition-examples";
 
-let encounterId: string;
-let encounterId2: string;
-let encounter: Encounter;
-let encounter2: Encounter;
-
-beforeEach(() => {
-  encounterId = faker.string.uuid();
-  encounterId2 = faker.string.uuid();
-  encounter = makeEncounter({ id: encounterId });
-  encounter2 = makeEncounter({ id: encounterId2 });
-});
+function initEncounter(practitionerId?: string, isIncludePractitioner = false) {
+  const encounterId = faker.string.uuid();
+
+  const encounter = isIncludePractitioner
+    ? makeEncounter({ id: encounterId }, { pract: practitionerId ?? faker.string.uuid() })
+    : makeEncounter({ id: encounterId });
+
+  return { encounter, encounterId };
```

Comment:
Can't the clients of this function just use `encounter.id`?

Actually, looking at this function and its clients, it seems its not needed?